src/lib/CMakeLists.txt

```diff
@@ -153,3 +153,39 @@ if(VISIT_PYTHON_SCRIPTING AND HAVE_LIBPYSIDE AND NOT PYSIDE_IN_PYTHON)
     SYMLINK_PYSIDE(SHIBOKEN_PYTHON_MODULE_DIR)
 endif()
 
+if(VISIT_PYTHON_SCRIPTING AND CONDUIT_DIR)
+    # symlink conduit python module so it works during dev builds
+    message(STATUS "Adding Conduit python module to lib/site-packages/")
+
+    if(NOT WIN32)
+        set(CONDUIT_PY_MODULE ${CONDUIT_DIR}/python-modules/conduit)
+        if(NOT EXISTS ${CONDUIT_PY_MODULE})
+            message(WARNING "Failed to find Conduit Python Module at ${CONDUIT_PY_MODULE}")
+        endif()
+        
+        execute_process(COMMAND ${CMAKE_COMMAND} -E remove -f
+                                ${CMAKE_CURRENT_BINARY_DIR}/site-packages/conduit
+                                 RESULT_VARIABLE CONDUIT_PY_SYM_REMOTE)
+        execute_process(COMMAND ${CMAKE_COMMAND} -E create_symlink
+                                ${CONDUIT_PY_MODULE}
+                                ${CMAKE_CURRENT_BINARY_DIR}/site-packages/conduit
+                                RESULT_VARIABLE CONDUIT_PY_SYM_CREATE)
+
+        if(NOT "${CONDUIT_PY_SYM_CREATE}" STREQUAL "0")
+            message(WARNING "Failed to create Conduit Python module symlink in lib/site-packages/")
+        endif()
+
+    else() # NOT WIN32
+        # correct lib/${CFG} directory
+        add_custom_target(conduit_python_module ALL
+            COMMAND ${CMAKE_COMMAND} -E copy_directory
```

Comment:
We have split logic for **NOT WIN32** where we symlink and otherwise use `-E copy_directory`. Any reason we shouldn't siimplify and use `-E copy_directory` everywhere? Yeah, it means the libs are copied but does that really matter here?

Reply:
Good Q: This logic follows other logic that uses the same approach. 

Copies should work (rpaths intact) and for conduit is really not many files. 

However for Python itself there can be a ton of files.  I think its nice to symlink if we can.